### ISSUE TITLE
Create Q&A docs by speaker from chats from ISC Online Spring Summit 2020

### ISSUE THREAD

**Issue by claredillon** [InnerSourceCommons/InnerSourceMarketing#7]
We have the chat files from the ISC Spring Summer Online Summit 2020. We need to pull out the questions from the docs and get the speakers to answer them. It will be an opportunity to send out their recordings again on social channels.

**Comment by claredillon** [comment]
Dmitrii created individual Q&A docs by speaker here: https://drive.google.com/drive/folders/1Lhmtg5Si9LlIavsFWM2b20awv0gjvMRb?usp=sharing

**Comment by lenucksi** [comment]
- Check with speakers if they have a blog and see if they want to post it there. 
- Otherwise postpone to until we have a blog. 

- Maybe consider the concept of blog aggregating, i.e. "planet innersourcecommons" ~ repost content from elsewhere.

- Podcast hosting, e.g. the Nearform Podcasts
- LinkedIn is limited to other people's contents, can't do articles there.

**Comment by lenucksi** [comment]
5+ documents exist, a very few to come.
Might depend on #10.

**Comment by spier** [comment]
@dsugrobov do you know what we will do with the "Q&A docs" once we have them? Just asking because it isn't obvious to me from the description in this issue.

**Comment by voborgus** [comment]
There are two options:

* Ask authors of the documents to put the Q&A on their's blog with the video from the summit.
* Wait for #24 and here is one of the prepared content.

**Comment by voborgus** [comment]
As for today, we have 8 documents with answers filled.

**Comment by spier** [comment]
Do you have the docs in our gDrive? Just curious to see how that content looks like.

My hunch is that #24 might take a bit.

So if we want to get this content out quickly, we would likely go with other options.

Btw how about just creating static documents for these Q&As and putting them up on innersourcecommongs.org? Would that be an alternative?

**Comment by voborgus** [comment]
Yep it`s [here](https://drive.google.com/drive/u/0/folders/1Lhmtg5Si9LlIavsFWM2b20awv0gjvMRb)

Since most documents are just short answers on comments during the talk they are valuable only together with the video itself in a blog post format. Now on the site, the only what we have is the link to a youtube playlist.

**Comment by spier** [comment]
@dsugrobov and Sebastian will post the first Q&A on the InnerSource Commons Group on LinkedIn.

ISC LinkedIn Group: https://www.linkedin.com/groups/4772921/

**Comment by spier** [comment]
Guilherme Dellagustin and Michael Graf from SAP have already published the Q&A from their session here:
https://blogs.sap.com/2020/05/08/growing-an-innersource-culture-at-sap-innersource-commons-summit/

We can model our own posts with the other Q&As after that if we like.

**Comment by spier** [comment]
rock'n'roll @dsugrobov's post is out 🚀 
https://www.linkedin.com/pulse/global-innersource-adoption-once-lessons-learned-dmitrii-sugrobov/

**Comment by spier** [comment]
I have an idea:
We convert the Q&A's into a simple format that the authors can then post "as is".
Assume that we can write a short instruction for how to post this to LinkedIn, it should take the authors/speakers 5 minutes max to post the Q&A.

I will try that for one post, so that you can see what I mean.

**Comment by spier** [comment]
Here an example of a template for a LinkedIn post that we could send to the speakers:
https://docs.google.com/document/d/1M0pvWjae-kk-5sf6e0lteYI4nUKcY9xYuXOCLMud5I8/edit#heading=h.uk11259ynvdy

**Comment by spier** [comment]
If we just went by the title "Create Q&A docs by speaker from chats from ISC Online Spring Summit 2020" then this issue would be done.

However I suggest that we go one step further, and send the instructions for posting to LinkedIn including a template for the post to all speakers. After that it is out of our hands (as the speakers have to post it themselves), and we can call this issue done.

Checkmark below means that instructions have been sent:

- [x] **publiished** Guilherme Dellagustin & Michael Graf - [article](https://blogs.sap.com/2020/05/08/growing-an-innersource-culture-at-sap-innersource-commons-summit/)
- [x] **published** Dmitrii Sugrobov - [article](https://www.linkedin.com/pulse/global-innersource-adoption-once-lessons-learned-dmitrii-sugrobov/)
- [x] **published** Isabel Drost-Fromm - [article](https://www.linkedin.com/pulse/dealing-growth-how-innersource-helps-isabel-drost-fromm/)
- [x] **published** Thomas Sadler - [article](https://www.linkedin.com/posts/thomas-sadler-5b7b0726_activity-6692374759432314880-gKQx/)
- [x] **published** Danese Cooper and Georg Grütter - [article](https://www.linkedin.com/pulse/evolution-innersource-commons-georg-gruetter/)
- [x] **published** Maximilian Capraro and Johannes Tigges - [article](https://www.linkedin.com/pulse/introduction-innersource-patterns-johannes-tigges/)
- [x] Danese Cooper
- [x] Jerry Tan

I collected the links of the published articles that I could find above. They should also be linked from https://www.linkedin.com/groups/4772921/

Sessions and speakers not listed here have not answered the Q&A. I suggest that we don't follow up on this any further, as I assume that Dmitrii has already tried all he could :)

**Comment by spier** [comment]
I am considering this work done.

All speakers that had filled out the Q&A have been provided with a template that they can use to publish a LinkedIn article of their talk and Q&A. Most of them have already done that (see above).

I think we have done what we could to use these Q&As for getting extra promotional value out of the last Summit. 

Hence I am closing this issue.
